### PR TITLE
Added option to load SQL file to the selected database

### PR DESCRIPTION
**Pull request by roman-fm**

### CODE REVIEW

File: src/Connection.php [hunk 1]
```diff
@@ -96,4 +95,36 @@ public function getDefaultStorageName() {
...
+
+    /**
+     * @param string $storageAlias
+     * @throws \InvalidArgumentException
+     */
+    protected function switchToStorageWithAlias($storageAlias) {
+        $this->switchToStorageWithName($this->getStorageNameByAlias($storageAlias));
+    }
+
+    protected function switchToDefaultStorage() {
```
Comment (hAdamm) [inline src/Connection.php]: Missing PHPDoc

File: src/Connection.php [hunk 2]
```diff
@@ -96,4 +95,36 @@ public function getDefaultStorageName() {
...
+        $this->switchToStorageWithName($this->getDefaultStorageName());
+    }
+
+    /**
+     * Import SQL dump from file to database with the given alias
+     * @param string $filename
+     * @param string $alias
+     * @return int count of sql commands
+     */
+    public function loadFileToStorageWithAlias($filename, $alias) {
```
Comment (hAdamm) [inline src/Connection.php]: '$alias' pls rename to '$storageAlias'

File: src/Connection.php (hunk 1 above)
Comment (roman-fm) [inline src/Connection.php]: Fixed

File: src/Connection.php (hunk 2 above)
Comment (roman-fm) [inline src/Connection.php]: Renamed

**Pull request merged**